### ISSUE TITLE
Epicondylitis

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10810]
**Preferred term label:**
Medial epicondylitis
Lateral epicondylitis
**Synonyms**

**Definition (free text, please give PubMed ID)**
Medial: Ultrasound imaging showing ECRB tendon thickening and hypoechoic areas with increased vascularity or MRI showing increased signal intensity or tear at the insertion site of the common flexor tendon
Lateral: Ultrasound imaging showing extensor carpi radialis brevis (ECRB) tendon thickening and hypoechoic areas with increased vascularity or MRI showing increased signal intensity or tear at the insertion site of the ECRB on the lateral aspect of the elbow

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Medial:
 Medial epicondylitis

Lateral:
Lateral epicondylitis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001364
- label: Lateral epicondylitis of the elbow
- parents: [Abnormality of the elbow (HP:0009811)](https://hpo.jax.org/app/browse/term/HP:0009811)
- pmid: PMID:28461918
- def: A chronic symptomatic degeneration of the wrist extensor tendons involving their attachment to the lateral epicondyle of the humerus, and is associated clinically with lateral elbow pain (tennis elbow). This term refers to the associated imaging finding. Ultrasound imaging showing extensor carpi radialis brevis (ECRB) tendon thickening and hypoechoic areas with increased vascularity or MRI showing increased signal intensity or tear at the insertion site of the ECRB on the lateral aspect of the elbow.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10810
- synonyms:

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001365
- label: Medial epicondylitis of the elbow
- parents: [Abnormality of the elbow (HP:0009811)](https://hpo.jax.org/app/browse/term/HP:0009811)
- pmid: PMID:26001427
- def: Medial epicondylitis, often referred to as golfer's elbow, is a common pathology. Flexor-pronator tendon degeneration occurs with repetitive forced wrist extension and forearm supination during activities involving wrist flexion and forearm pronation. This term refers to the associated imaging finding. Ultrasound imaging showing ECRB tendon thickening and hypoechoic areas with increased vascularity or MRI showing increased signal intensity or tear at the insertion site of the common flexor tendon.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10810
- synonyms: Golfer's elbow(NONE)
